game_controller.py

```diff
@@ -206,7 +206,6 @@ def _apply_pre_actions(self) -> None:
 
     def _disable_used_player_actions(self):
```

Comment:
Do we need this method? 
Probably, it's better use
`self._city.active_player.remove_used_card()`
directly